product_docs/docs/pem/9/pem_rel_notes/920_rel_notes.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Version 9.2.0"
+---
+
+New features, enhancements, bug fixes, and other changes in PEM 9.2.0 include:
+
+| Type        | Description                                                                                      |
+| ----------- | -------------------------------------------------------------------------------------------------|
+| Bug fix     | Fixed the `Replication Time Lag` graph to display the status as stopped if the server is down/offline on the Streaming Analysis dashboard.[Support ticket: 91220] |
+| Bug fix     | Removed the agent status from Failover Manager Dashboard under `efm_cluster_node_status`.[Support ticket: #1228607] |
+| Bug fix     | Fixed an `unknown error` while updating/creating the server from the PEM web client. |
+| Bug fix     | Fixed an issue to return a valid message for invalid parameters and removed unwanted parameters in PUT server REST API. |
+| Bug fix     | Updated cryptography package to the secured version. |
+| Bug fix     | Updated webpack(NPM) package to the secured version. |
+| Bug fix     | Fixed an error caused by an increase/decrease in the number of parameters when changing the alert templates. |
+| Bug fix     | Fixed the `Static object initialization fiasco` issue caused while initializing the BARMAN singleton object at the startup. |
+| Bug fix     | Resolved the error `Login Group/Roles 500 error: UNION types char and text cannot be matched`. This error was displaying while trying to open the dependents tab for any role with PotgreSQL or Advanced Server 15 as the backend server. |
+| Bug fix     | Provided helpful message to the user instead of default Python error if a token is invalid. [Support ticket: #89276] |
+| Bug fix     | Fixed extension name warning while creating a probe. Even after providing a valid extension name. |
+| Bug fix     | Fixed the error in the `--config-file` switch while trying to register the Barman server in PEM using the command line interface. |
+| Bug fix     | Fixed documentation of replacing PEM self-signed SSL certificates with trusted third-party CA certificates. |
+| Bug fix     | Fixed the missing favicon icon for help docs and API link. |
+| Bug fix     | Fixed the warning message while deleting a folder in the `Manage Template` section of `Capacity Manager`. |
+| Bug fix     | Fixed the tunning wizard report where the number of servers selected didn't match the servers displayed in the `GoTo` dropdown list. |
+| Bug fix     | Fixed the default time as `Now` at the end of the sample set in the performance diagnostics.
```

Comment:
```suggestion
| Enhancement     | The default sample end time for performance diagnostics is now `Now` rather than the time the window was first opened, meaning the data will update upon clicking 'Run'.
```